src/components/ActiveModule/index.vue

```diff
@@ -245,8 +245,10 @@ export default {
     enabled: {
       get() {
         if (!this.moduleName) return false
-        return this.$store.state.modVModules.active[this.moduleName].meta
-          .enabled
+        return (
+          this.$store.state.modVModules.active[this.moduleName] &&
+          this.$store.state.modVModules.active[this.moduleName].meta.enabled
```

Comment:
And this change?

Reply:
I think I was relying on async timing(/race condition) before the change to the module order change, but I'm not 100% sure on this.

Both this and the other Vue file change produced errors after I made the module order change as they were trying to access data which is now not present yet.

Tbh both of those Vue files need a complete rewrite, they're full of terrible code.